src/components/profile/Profile.ts

```diff
@@ -1,89 +1,97 @@
+      return html`
+        <div class="contact-item">
+          <md-badge circle color="violet">
+            <md-icon name="icon-email-active_12" size="8"></md-icon>
+          </md-badge>
+          <span>${this.presetTags.email}</span>
+        </div>
+      `;
+    }
 
-  getTopContent() {
-    return html`
-      <section class="top-content">
-        <md-avatar
-          .title="${this.presetTags["name"].join(" ")}"
-          alt=${this.presetTags["name"].join(" ")}
-          src=${this.profile.picture || undefined}
-          .size=${48}
-        ></md-avatar>
-        <h5 title="Name" class="customer-name">
-          ${this.presetTags["name"].join(" ")}
-        </h5>
-        <h5 title="Label" class="customer-label">
-          ${this.presetTags["label"] ? this.presetTags["label"].join(" ") : "VIP Customer"}
-        </h5>
-        ${this.contactItem()}
-      </section>
-    `;
-  }
+    getTopContent() {
+      return html`
+        <section class="top-content">
+          <md-avatar
+            .title="${this.presetTags["name"].join(" ")}"
+            alt=${this.presetTags["name"].join(" ")}
+            src=${this.profile.picture || undefined}
+            .size=${48}
+          ></md-avatar>
+          <h5 title="Name" class="customer-name">
+            ${this.presetTags["name"].join(" ")}
+          </h5>
+          <h5 title="Label" class="customer-label">
+            ${this.presetTags["label"] ? this.presetTags["label"].join(" ") : "VIP Customer"}
+          </h5>
+          ${this.contactItem()}
+        </section>
+      `;
+    }
 
-  getTable() {
-    return html`
-      <table title="Profile Details">
-        ${this.profile
-          .filter((x: any) => x.query.type === "table")
-          .map((x: any) => {
-            return html`
-              <tr>
-                <td class="title">${x.query.DisplayName}</td>
-                <td class="value">${this.getValue(x)}</td>
-              </tr>
-            `;
-          })}
-      </table>
-    `;
-  }
-  getValue(x: any) {
-    let result = null;
+    getTable() {
+      return html`
+        <table title="Profile Details">
+          ${this.profile
+            .filter((x: any) => x.query.type === "table")
+            .map((x: any) => {
+              return html`
+                <tr>
+                  <td class="title">${x.query.DisplayName}</td>
+                  <td class="value">${this.getValue(x)}</td>
+                </tr>
+              `;
+            })}
+        </table>
+      `;
+    }
+    getValue(x: any) {
+      let result = null;
 
-    if (x.query.formatValue) {
-      try {
-        result = x.result.map(x.query.formatValue).join(", ");
-      } catch (err) {
-        console.log("Unable to format value", err);
+      if (x.query.formatValue) {
+        try {
+          result = x.result.map(x.query.formatValue).join(", ");
+        } catch (err) {
+          console.log("Unable to format value", err);
```

Comment:
if this is not temporaty console log, add a little more  context, e.g. prefix to tell the developer where this message is coming from. 
```
CJAAS Profile:
```